mrp_subcontracting_resupply_link/models/purchase_order.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import SUPERUSER_ID, _, api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    mrp_resupply_picking_count = fields.Integer(
```

Comment:
Change all for being more exact, and at the same time, more concise:

```suggestion
    subcontracting_resupply_count = fields.Integer(
```